ionic/platform/storage/storage.ts

```diff
@@ -27,21 +27,28 @@ export class Storage {
       return null;
     }
   }
-  set(key, value) {
+  set(key: string, value: any) {
     return this._strategy.set(key, value);
   }
-  remove(key) {
+  remove(key: string) {
     return this._strategy.remove(key);
   }
-  query(query, params) {
+  query(query: string, params: any) {
```

Comment:
params is a optional param, please consider changing it to params?
this way users don't get warning when running queries like `SELECT * FROM users;`